core/server/update-check.js

```diff
@@ -38,6 +38,24 @@ var crypto   = require('crypto'),
     checkEndpoint = 'updates.ghost.org',
     currentVersion = config.ghostVersion;
 
+/**
+ * Attempts to parse string to JSON.
+ * @return {Object} is successful. Otherwise, undefined.
+ */
+function safeParseJson(json) {
```

Comment:
@vkandy Just came across https://gist.github.com/gucheen/12b90451f04733078d7f which shows a more concise Lodash way of safely parsing JSON - maybe something to consider?